hack/verify-typecheck.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OPENSHIFT_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OPENSHIFT_ROOT}/hack/lib/init.sh"
+
+os::golang::verify_go_version
+
+cd "${OPENSHIFT_ROOT}"
+
+platforms=${PLATFORMS:-'linux/amd64,darwin/amd64,linux/arm,linux/386,linux/arm64,linux/ppc64le,linux/s390x,darwin/386'}
```

Comment:
You're missing windows here and that is an official release we care about: https://github.com/openshift/origin/releases/ At the same time we could limit this to 64-bit binaries, since these are the only ones we ship.

Reply:
crio breaks windows that's why I removed it

Reply:
how are we shipping windows binaries, then?

Reply:
we ship client tools only, right?
```
ERROR(windows) pkg/build/builder/crioclient/crioclient.go:14:30: RawSockaddrUnix not declared by package syscall
ERROR(windows) pkg/build/builder/crioclient/crioclient.go:14:30: invalid operation: (syscall.RawSockaddrUnix literal) (value with invalid type) has no field or method Path
```

Reply:
ah. can we fix that with the appropriate build tags? IIRC the linux cli uses gssapi stuff, obviously not possible on windows